clear-site-data/support/server.py

```diff
@@ -0,0 +1,11 @@
+import json
```

Comment:
Nit: We usually call files like this something like `echo-clear-site-data.py`, just so that the intent is clear.